File: Project/Backend/src/ArtAuction.Application/DTOs/PostSold/PostSoldDto.cs
using ArtAuction.Application.DTOs.ArtworkPost;

namespace ArtAuction.Application.DTOs.PostSold;

public class PostSoldDto
{
    // Attributes
    public ArtworkPostDto ArtworkPost { get; set; }
    public string BuyerName { get; set; }
    public string FinalPrice { get; set; }
    public bool IsPaid { get; set; }
}

File: AuctionServices/src/main/java/com/artauction/auctionservice/Dtos/PostSold/PostSoldDto.java
package com.artauction.auctionservice.Dtos.PostSold;

import com.artauction.auctionservice.Dtos.ArtworkPost.ArtworkPostDto;
import lombok.Getter;
import lombok.Setter;

import java.math.BigDecimal;

@Getter
@Setter
public class PostSoldDto {
    private ArtworkPostDto artworkPost;
    private String buyerName;
    private BigDecimal finalPrice;
    private boolean isPaid;
}
